code/modules/power/smes.dm

```diff
@@ -64,9 +64,10 @@
 					if(term && term.dir == turn(d, 180))
 						terminal = term
 						break dir_loop
-		if(!terminal)
+		/*if(!terminal)
 			stat |= BROKEN
 			return
+			*/ //

```

Comment:
I think the `//` is redundant here.

Reply:
Of course it is. I forgot to remove it. It was there to get check this later but then i only deleted the marker